boot/zephyr/main.c

```diff
@@ -69,7 +70,8 @@ void main(void)
 	irq_lock();
 	_MspSet(vt->msp);
 
-	BOOT_LOG_INF("Jumping to the first image slot");
+	BOOT_LOG_INF("Disabling clock and Jumping to the first image slot");
+	sys_clock_disable();
```

Comment:
What do you think about adding a "(void)irq_lock();" here similar to how sys_reboot() works: https://gerrit.zephyrproject.org/r/gitweb?p=zephyr.git;a=blob;f=misc/reboot.c;h=864aa0dd5ba7fe428e1531302df1b3800a3fa1c4;hb=refs/heads/master#l20